doc/libalsaaudio.rst

```diff
@@ -410,22 +410,15 @@ PCM objects have the following methods:
 
    *New in 0.10*
 
-.. method:: PCM.close()
```

Comment:
if this is legit (is it?), then it definitely belongs into a separate commit.

Reply:
It's legit. `close` wasn't deleted, but moved to the end. Creating a separate commit for that seems overkill to me, though.

Reply:
> It's legit. close wasn't deleted, but moved to the end.

yes, i've seen that later.

> Creating a separate commit for that in retrospect seems overkill to me, though.

but this is exactly what good commit hygiene is about. it would have avoided me asking that "stupid" question.

and fixing it would take about 30 seconds in git gui (plus however much you need to type the commit message).